### PR TITLE
Some historical overlooked Ubuntu-assigned CVEs

### PR DESCRIPTION
**Pull request by setharnold**
Hello, here's a first run through of some of the CVEs that Ubuntu has assigned. Our apologies for letting these become stale.

I believe we still have these CVEs outstanding, they will come in a future pull request:

CVE-2012-0945
CVE-2012-0951
CVE-2012-0952
CVE-2012-0953
CVE-2012-0955
CVE-2013-1053
CVE-2013-1054
CVE-2013-1055
CVE-2014-1420
CVE-2014-1422
CVE-2014-1423
CVE-2015-7946

Thanks

### CODE REVIEW

**Comment by cve-team** [comment]
setharnold is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

File: 2011/1xxx/CVE-2011-1830.json [hunk 1]
```diff
@@ -1,18 +1,92 @@
...
+      }
+   ],
+   "data_format": "MITRE",
+   "data_type": "CVE",
+   "data_version": "4.0",
+   "description": {
+      "description_data": [
+         {
+            "lang": "eng",
+            "value": "Ekiga attempted to load a module from /tmp/ekiga_test.so."
```
Comment (cve-team) [inline 2011/1xxx/CVE-2011-1830.json]: The affected version is not included in the Description.

File: 2011/3xxx/CVE-2011-3145.json [hunk 2]
```diff
@@ -1,18 +1,101 @@
 {
```
Comment (cve-team) [inline 2011/3xxx/CVE-2011-3145.json]: CNA(s) of user (canonical) do not match owning CNA.

File: 2011/3xxx/CVE-2011-3147.json [hunk 3]
```diff
@@ -1,18 +1,92 @@
 {
```
Comment (cve-team) [inline 2011/3xxx/CVE-2011-3147.json]: CNA(s) of user (canonical) do not match owning CNA.

File: 2011/3xxx/CVE-2011-3151.json [hunk 4]
```diff
@@ -1,18 +1,92 @@
 {
```
Comment (cve-team) [inline 2011/3xxx/CVE-2011-3151.json]: CNA(s) of user (canonical) do not match owning CNA.

File: 2011/1xxx/CVE-2011-1830.json (hunk 1 above)
Comment (setharnold) [reply 2011/1xxx/CVE-2011-1830.json]: Is there a reason to duplicate the affected version information from the specific version fields? Should this be duplicated in all the descriptions?

```
$ jq '.affects.vendor.vendor_data[0].product.product_data[0].version' 2011/1xxx/CVE-2011-1830.json
{
  "version_data": [
    {
      "affected": "<",
      "version_value": "3.3.0"
    }
  ]
}
```

Thanks

File: 2011/3xxx/CVE-2011-3145.json (hunk 2 above)
Comment (setharnold) [reply 2011/3xxx/CVE-2011-3145.json]: I'm sorry, I don't understand this issue. `security@ubuntu.com` is a stable email address for the Ubuntu Security Team that will continue to be useful even after I leave the company. (Furthermore, I wasn't with the company in 2011, so the number was actually assigned by someone else.)

What resolution is expected here?

Thanks

**Comment by iamleot** [comment]
Hello Seth,

Seth Arnold writes:
> Is there a reason to duplicate the affected version information from the specific version fields? Should this be duplicated in all the descriptions?

Yes, not all interfaces exposes the affected version information,
e.g.  by following CVE via RSS/Atom feeds.  Similarly IIRC on
nvd.nist.gov the CPE information is "hidden" by default.  Having
them in the description field permits to quickly look at them.

File: 2011/1xxx/CVE-2011-1830.json (hunk 1 above)
Comment (cve-team) [reply 2011/1xxx/CVE-2011-1830.json]: Hello Seth,

The information provided in the fields are not all reflected when the entry is published to cve.mitre.org. Though the abundant information provided in JSON is beneficial to down stream users that ingest the feed, we still need to make sure the right information is present in the website at publication. This is why we ask that the affects field information be present in the description along with the problem type /  impact.

Thanks for the question and understanding,
The CVE Assignment Team

File: 2011/3xxx/CVE-2011-3145.json (hunk 2 above)
Comment (cve-team) [reply 2011/3xxx/CVE-2011-3145.json]: It looks like these IDs were not assigned to Canonical/Ubuntu.  We will be happy to incorporate the provided information but we must manually review these Ids.

Thanks.

File: 2011/1xxx/CVE-2011-1830.json (hunk 1 above)
Comment (cve-team) [reply 2011/1xxx/CVE-2011-1830.json]: @setharnold just pinging to see if you have had a chance to review.

**Comment by setharnold** [comment]
On Thu, Apr 11, 2019, 07:40 CVE Team <notifications@github.com> wrote:

> @setharnold <https://github.com/setharnold> just pinging to see if you
> have had a chance to review
>

My apologies, this isn't forgotten, just a bit behind other deadlines.

Thanks

>

File: 2011/3xxx/CVE-2011-3145.json (hunk 2 above)
Comment (cve-team) [inline 2011/3xxx/CVE-2011-3145.json]: CNA(s) of user (canonical) do not match owning CNA.

File: 2011/3xxx/CVE-2011-3147.json (hunk 3 above)
Comment (cve-team) [inline 2011/3xxx/CVE-2011-3147.json]: CNA(s) of user (canonical) do not match owning CNA.

File: 2011/3xxx/CVE-2011-3151.json (hunk 4 above)
Comment (cve-team) [inline 2011/3xxx/CVE-2011-3151.json]: CNA(s) of user (canonical) do not match owning CNA.

**Comment by setharnold** [comment]
I've updated the descriptions to be more human-friendly.

I left the CVE numbers as-is, because most of them have been used in USNs already.

Thanks

**Pull request merged**